x-pack/plugins/cloud_security_posture/common/types/rules/v5.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema, TypeOf } from '@kbn/config-schema';
+
+export type {
+  cspBenchmarkRuleMetadataSchema,
+  CspBenchmarkRuleMetadata,
+  cspBenchmarkRuleSchema,
+  CspBenchmarkRule,
+  FindCspBenchmarkRuleResponse,
+} from './v3';
+export type {
+  PageUrlParams,
+  rulesToUpdate,
+  CspBenchmarkRulesBulkActionRequestSchema,
+  CspBenchmarkRulesBulkActionResponse,
+  RuleStateAttributes,
+  CspBenchmarkRulesStates,
+  cspSettingsSchema,
+  CspSettings,
+  BulkActionBenchmarkRulesResponse,
+} from './v4';
+
+const DEFAULT_BENCHMARK_RULES_PER_PAGE = 25;
```

Comment:
I'm not sure why this or any other instances of this value are kept here and are basically versioned. i dont see a reason